source/gui/settingsDialogs.py

```diff
@@ -352,13 +352,15 @@ class SettingsPanel(
 
 	To use this panel:
 		* Set L{title} to the title of the category.
+		* Set l{panelDescription} describe the purpose of the category.
```

Comment:
We use [RST](https://devguide.python.org/documentation/markup/).
Please see the [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md) for more information.

```suggestion
		* Set :attr:`panelDescription` describe the purpose of the category.
```